recipes-core/zsh/files/profile_d_10-zshfix

```diff
@@ -4,5 +4,5 @@ if [[ "$ZSH_NAME" = sh ]]
 then
 	# zsh shall be used as /bin/zsh for login-shells
 	# we still need a sane default prompt (without expansion)
-	export PS1="sh@CAROS: # "
+	export PS1="$USER@$(hostname): # "
```

Comment:
using environment variable is fine. calling a process on every shell/profile is not ideal. is $HOST not available?